app/forms/apply_applications/trainee_data_form.rb

```diff
@@ -36,7 +36,7 @@ def save
       trainee.progress.diversity = true
       trainee.progress.degrees = true
       trainee.progress.trainee_data = true
-      Trainees::Update.call(trainee: trainee)
+      Trainees::Update.call(trainee: trainee, set_academic_cycles_now: true)
```

Comment:
what is `set_academic_cycles_now: true` doing?

Reply:
Its causing the update method to run perform_now rather than perform_later on the start academic cycles job (meaning we update the academic cycle immediately rather than add it to the overnight jobs)